### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by adumortier**
## Who worked on this

- [x] Alexis
- [ ] Maria
- [ ] Ed
- [ ] Cody

## Type of change

- [ ] Style (Non-breaking Style code)
- [x] Testing (Non-breaking Testing code)
- [ ] Bug fix or refactor (change which fixes an issue)
- [ ] New feature (change which adds functionality)

## Summary of the change.

- added Travis CI setup

## Checklist:

- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code if necessary

### CODE REVIEW

File: tests/test_at_risk_user.py [hunk 1]
```diff
@@ -27,25 +27,25 @@ def tearDown(self):
...
+    response = self.test_app.post('/atriskuser/4321', json={ "name": "Alexis", "address": "125 ocean ave", "city": "Denver", "state": "ca", "zipcode": "80203", "phone_number": "123-456-7890"})
     payload = json.loads(response.data)
     self.assertEqual(response.status_code, 201)
     self.assertEqual(payload['data']['attributes']['name'], 'Alexis')
     self.assertEqual(payload['data']['attributes']['at_risk_user_id'], '4321')
     self.assertEqual(payload['data']['attributes']['address'], '125 ocean ave')
     self.assertEqual(payload['data']['attributes']['city'], 'Denver')
     self.assertEqual(payload['data']['attributes']['state'], 'ca')
-    self.assertEqual(payload['data']['attributes']['zip code'], '91763')
+    self.assertEqual(payload['data']['attributes']['zip code'], '80203')
```
Comment (adumortier) [inline tests/test_at_risk_user.py:38]: change the test with a `zipcode` that will return a Kroger store

File: trackbasket_be/models/krogerservice.py [hunk 2]
```diff
@@ -47,15 +47,17 @@ def closest_store(cls, zipcode):
...
       return  { 'error': 'no store found for this zipcode' }
 
     blacklist = ['shell company']
     i = 0
     while (parsed_response['data'][i]['name'].lower() in blacklist) and (i < number_results - 1):
       i += 1
-    if (parsed_response['data'][i]['name'] in blacklist):
+    
+    
+    if (parsed_response['data'][i]['name'].lower() in blacklist):
```
Comment (adumortier) [inline trackbasket_be/models/krogerservice.py:60]: takes care of the case where all results returned belong to the blacklist

File: tests/test_store.py [hunk 3]
```diff
@@ -27,19 +27,19 @@ def test_new_store(self):
...
-  #   self.assertEqual(payload['latitude'], 39.6951161)
-  #   self.assertEqual(payload['longitude'], -104.8109163)
+    self.assertEqual(payload['location_id'], '62000010')
+    self.assertEqual(payload['name'], 'KINGSOOPERS')
+    self.assertEqual(payload['address'], '15250 E Mississippi Ave')
+    self.assertEqual(payload['city'], 'Aurora')
+    self.assertEqual(payload['state'], 'CO')
+    self.assertEqual(payload['zipcode'], '80012')
+    self.assertEqual(payload['latitude'], 39.6951161)
+    self.assertEqual(payload['longitude'], -104.8109163)
```
Comment (mronauli) [inline tests/test_store.py:42]: Nice test for a store that exists!

File: trackbasket_be/models/krogerservice.py (hunk 2 above)
Comment (mronauli) [reply trackbasket_be/models/krogerservice.py:60]: would be awesome to see a test for this!

File: tests/test_at_risk_user.py [hunk 4]
```diff
@@ -27,25 +27,25 @@ def tearDown(self):
...
     self.assertEqual(payload['data']['attributes']['phone number'], '123-456-7890')
 
-    response = self.test_app.post('/atriskuser/4321', json={ "name": "Alexis", "address": "125 ocean ave", "city": "Denver", "state": "ca", "zipcode": "91763", "phone_number": "123-456-7890"})
+    response = self.test_app.post('/atriskuser/4321', json={ "name": "Alexis", "address": "125 ocean ave", "city": "Denver", "state": "ca", "zipcode": "80203", "phone_number": "123-456-7890"})
     self.assertEqual(response.status_code, 400)
```
Comment (mronauli) [inline tests/test_at_risk_user.py:42]: nice edge case

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+language: 
+  - python
+
+python:
+  - 3.7
+
+install: 
+  - pip install -r requirements.txt
```
Comment (mronauli) [inline .travis.yml:8]: installs dependencies
